File: src/2833.furthest-point-from-origin.3019/solution.ts
function furthestDistanceFromOrigin (moves: string): number {
    let diff = 0;
    let _ = 0;
    for (let i = 0; i < moves.length; i++) {
        if (moves[i] === 'L') {
            diff++;
        } else if (moves[i] === 'R') {
            diff--;
        } else {
            _++;
        }
    }

    return Math.abs(diff) + _;
}

File: src/2833.furthest-point-from-origin.3019/solution.java
class Solution {
    public int furthestDistanceFromOrigin(String moves) {
        int L = 0;
        int R = 0;
        int U = 0;

        for (char c : moves.toCharArray()) {
            if (c == 'L') {
                L++;
            } else if (c == 'R') {
                R++;
            } else {
                U++;
            }

        }
        return Math.abs(L - R) + U;
    }
}